src/KernelClisStrategy.sol

```diff
@@ -38,4 +41,17 @@ contract KernelClisStrategy is KernelStrategy {
         //wrap native token
         IWBNB(asset_).deposit{value: assets}();
     }
+
+    function _availableAssets(address asset_) internal view virtual override returns (uint256 availableAssets) {
+        availableAssets = IERC20(asset_).balanceOf(address(this));
+
+        StrategyStorage storage strategyStorage = _getStrategyStorage();
+
+        if (strategyStorage.syncWithdraw && asset_ == asset()) {
+            IStakerGateway stakerGateway = IStakerGateway(strategyStorage.stakerGateway);
+            address clisbnb = IKernelConfig(stakerGateway.getConfig()).getClisBnbAddress();
+            uint256 availableAssetsInKernel = stakerGateway.balanceOf(clisbnb, address(this));
```

Comment:
use the rate provider here to convert clisBNB to BNB

this is essentially hardcoding a rate of 1

Reply:
fixed in [5cea29d](https://github.com/yieldnest/yieldnest-kernel-lrt/pull/63/commits/5cea29d4ebca661b47de1b8c77806c367e02be63)